binding/python3/CMakeLists.txt

```diff
@@ -62,5 +62,5 @@ configure_file(${CMAKE_CURRENT_SOURCE_DIR}/_version.py.i ${CMAKE_CURRENT_BINARY_
 INSTALL(TARGETS _${PROJECT_NAME}
     DESTINATION ${PYTHON_INSTALL_DESTINATION}
 )
-install(FILES ${CMAKE_CURRENT_BINARY_DIR}/_version.py DESTINATION ${PYTHON_INSTALL_DESTINATION}/${PROJECT_NAME})
```

Comment:
Actually, I think this is right, I had to manually move _version.py from site-packages to site-packages/ezc3d, both in Windows and Debian, miniconda3, SWIG 3 and 4 both.